api/shape/Box.js

```diff
@@ -31,7 +31,7 @@ Shape.registerReifier('Box', (geometry) => {
   ]);
   const b = a.fill();
   const c = b.ex(top, bottom);
-  const d = c.orient({
+  const d = c.orient2({
```

Comment:
Is the "2" here meant to read like "To"?

Reply:
Actually, it's just because I couldn't deprecate it quite yet. :)

I'll remove it in a while